frontend/src/Order.tsx

```diff
@@ -178,30 +190,26 @@ const Order = () => {
                     <Grid item container xs={12} spacing={3} alignItems="center">
                         <Grid item xs={10} sm={5} data-testid={'ex_dropdown'}>
                             <FieldHeader headerText={'EX classification'} />
-                            <SearchableDropdown options={exClassOptions} onSelect={item => setSingleField('exClass', item.title)} />
+                            <Select options={exClassOptions} onSelect={item => setSingleField('exClass', item.title)} />
                         </Grid>
                         <HelpIcon helpText={'info text'} />
                     </Grid>
                     <Grid item container xs={12} spacing={3} alignItems="center">
-                        {exClass !== '' ? ( //Show accessories when EX-class is chosen. TODO: different preselected depending on EXClass
-                            <Grid item xs={10} sm={5} data-testid={'accessories_dropdown'}>
-                                <FieldHeader headerText={'Accessories'} />
-                                <AccessorySelector selectedAccessories={accessories} setSingleField={setSingleField} />
-                            </Grid>
-                        ) : (
-                            <></>
-                        )}
+                        <Grid item xs={10} sm={5} data-testid={'accessories_dropdown'}>
+                            <FieldHeader headerText={'Accessories'} />
+                            <AccessorySelector selectedAccessories={accessories} setSingleField={setSingleField} />
+                        </Grid>
+                    </Grid>
+                    <Grid item container xs={12} spacing={3} alignItems="center">
+                        <Grid item xs={10} sm={5} data-testid={'user_type_dropdown'}>
+                            <FieldHeader headerText={'User type'} />
+                            <Select options={userTypeOptions} onSelect={item => setSingleField('userType', item.title)} />
+                        </Grid>
+                        <HelpIcon helpText={'info text'} />
                     </Grid>
                     {userType === 'Equinor personnel' ? (
                         //Equinor employee device
                         <>
```

Comment:
We don't need the parent `<></>` anymore as the outer `Grid `contains all element